app/src/main/java/com/ccc/nameapp/scenes/login/main/MainLoginFragment.kt

```diff
@@ -88,7 +88,10 @@ class MainLoginFragment : DaggerFragment(), MainLoginView {
     }
 
     private fun handleEvents() {
-        mView.nextView.setOnClickListener { onNextClick() }
+        mView.nextView.setOnClickListener {
+            throw RuntimeException("Test Crash") // Force a crash
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/daolq3012/Structure_Android/blob/735598970d3909e548031fb495a8926767753d8a/app/src/main/java/com/ccc/nameapp/scenes/login/main/MainLoginFragment.kt#L92"></span>This expression is followed by unreachable code which should either be used or removed.</td>
    </tr>
  </tbody>
</table>